clumpr/HistogramAlg.py

```diff
@@ -41,15 +41,20 @@
 
 
 def intersect_cost(event):
-    """The cost added if both reconciliations being looked at share a particular event"""
+    """
+    The cost added if both reconciliations being looked at share a particular event
+    :param event <tuple>   - the event being shared
+    :return <int>          - the cost
+    """
     return 0
 
 
 def cost(event, zero_loss):
-    """The cost added if exactly one of the reconciliations being looked at share a particular event.
-    :param event:       The event being shared
-    :param zero_loss:   Whether loss events should have cost = 0
-    :return:            The cost
+    """
+    The cost added if exactly one of the reconciliations being looked at share a particular event
+    :param event <tuple>      - the event being shared
```

Comment:
Do you know what this docstring format is called? We might want to put it in the readme or other files that tells other people how to contribute to the repo.